data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3285 @@
+			choice
+				`	"Seems good to me, I will do it."`
+					accept
+				`	"What kind of ship are we talking about?"`
+				`	"Wait you mean they could shoot me? No way, I am out!"`
+					decline
+			`	She seems anxious to tell you. "It is a human ship, and quite a big one. We do not know its designation in human space but it carries a few small ships, and uses alien technologies. You do not need to be able to destroy it, just scan it and then come back here." She did not seem to want to answer the last question, and maybe only answered this one so you would go away. Now she is waiting for an answer, maybe you should reassure her in your abilities before she starts doubting you, and hiring someone else.`
+			choice
+				`	"I am not worried about any one ship, I will do it."`
+					accept
+				`	"That seems too dangerous to me."`
+					decline
+	
+	on enter "Hi Yahr"
+		dialog `You see the ship as you enter the system, it is a navy Auxiliary. Whatever happens, the Unfettered asked you not to destroy it and only scan it, at least for now.`
+	
+	npc save "scan cargo"
+		government "Alpha"
+		personality target staying timid entering
+		system "Hi Yahr"
+		ship "Auxiliary (Alpha)" "Settler"
+		
+		conversation
+			`You have finished scanning the Auxiliary. There can be no doubt, this is has Alphas on it. As you prepare yourself to exit the system and bring the logs back, you receive a message from the local Unfettered authorities.`
+			`	"We demand that you land right away. If you go to any other system that ship may pursue you. We do not want a dispute between our suppliers of jump drives; this is probably just a misunderstanding. If you land we can guarantee your safety here, we have requested that ship to land on another port, on Firelode, since it did fire on you first you without any apparent provocation."`
+			choice
+				`	"I will land as you ask."`
+					accept
+				`	"Screw you and your government I am taking that ship down!"`
+			`	"Consider this carefully before taking any actions you may regret in the future. Our offer still stands."`
+	
+	npc
+		government "Alpha"
+		personality heroic nemesis staying vindictive
+		system "Hi Yahr"
+		ship "Far Osk 27" "Prancer"
+		ship "Far Osk 27" "Vixen"
+		ship "Far Osk 27" "Anger"
+		ship "Far Osk 27" "Dart"
+	
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on complete
+		dialog `You are offered some very good beverages to keep you waiting, even by your standards. They probably know of your mission. After a while, the Alpha ship goes away, you can now go back to <origin> with the scanner logs.`
+
+
+
+mission "Unfettered: Solifuge 3: Auxiliary Design Logs"
+	name `Bring back the scanner logs`
+	description `Now that you have scanned the Alpha ship, return to <destination> to get your reward.`
+	source "Warfeed"
+	landing
```

Comment:
I think Amazinite said something about landing have to be at the top above everything. I might've missed a few before this as well.

Reply:
Oh so there is actually a rule about the order in which it should be, didn't know about that so I made my custom order, will change it